test/test-v8debugapi.js

```diff
@@ -307,6 +307,31 @@ describe('v8debugapi', function() {
       });
     });
 
+    it('should resolve correct frame count', function(done) {
+      // clone a clean breakpointInFoo
+      var bp  = {id: breakpointInFoo.id, location: breakpointInFoo.location};
+      var oldCount = config.capture.maxExpandFrames;
+      config.capture.maxExpandFrames = 0;
+      api.set(bp, function(err) {
+        assert.ifError(err);
+        api.wait(bp, function(err) {
+          assert.ifError(err);
+          assert.ok(bp.stackFrames);
+          assert.ok(bp.variableTable);
+
+          var topFrame = bp.stackFrames[0];
+          assert.ok(topFrame);
+          assert.equal(topFrame['function'], 'foo');
+          assert.equal(topFrame.arguments.length, 0);
```

Comment:
Instead of an empty list, I wonder if we can provide an error StatusMessage indicating that the argument and local list has been omitted. How hard would this be?